acceptancetests/assess

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python33
```

Comment:
Typo! This should be python3.

Reply:
Whoops... Forgot to properly terminate my find and replace regex :D